vue/components/ui/molecules/filter/filter.vue

```diff
@@ -321,6 +341,15 @@ export const Filter = {
                 return false;
             }
 
+            // in case of an autorefresh you want to get at least
+            // the same amount of items that are currently being shown
+            if (auto) {
+                this.$emit("auto:refresh:start");

```

Comment:
Why is this event needed?
It seems is because of some loading, but we already have the `update:loading` event for those things